src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,18 @@ title = "Releases"
 weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
+## v0.14.1
+
+This is a bugfix release:
+
+* Stop using cluster-owned tag for AWS Security Group lookup.
+* Running the `subctl diagnose firewall` command with individual kubeconfigs will now override the default namespace to `submariner-operator`.
+* Fix issues with gateway public IPs resolver.
+* The `subctl diagnose kube-proxy-mode` command now also checks if IPVS is enabled or not.
```

Comment:
The subctl diagnose kube-proxy-mode already supported the IPVS validation. The issue was seen with downstream container images where the iproute package returns a different error message.

```suggestion
* The `subctl diagnose kube-proxy-mode` command now works fine with different versions of iproute packages.
```